js/parser.js

```diff
@@ -283,25 +277,144 @@ var ERMrest = (function(module) {
          * @returns {String} the string representation of the Location, which is the full URI.
          */
         toString: function(){
-            return this._uri;
+            return this.uri;
         },
 
 
         /**
-         *
+         * <service>/catalog/<catalogId>/<api>/<projectionSchema:projectionTable>/<filters>/<joins>/<search>/<sort>/<page>/<queryParams>
          * @returns {String} The full URI of the location
```

Comment:
How does this URI differ from `ermrestUri`? Could you mention that in the comments here.